packages/backend-next/src/index.ts

```diff
@@ -45,4 +46,7 @@ backend.add(import('@backstage/plugin-search-backend/alpha'));
 backend.add(import('@backstage/plugin-techdocs-backend/alpha'));
 backend.add(import('@backstage/plugin-todo-backend'));
 
+// Sonarqube
+backend.add(sonarqubePlugin());
```

Comment:
This is not how plugins are imported any longer, if you look at the example above this should be 
```suggestion
backend.add(import('@backstage/plugin-sonarqube-backend');
```